test/test_parser.cpp

```diff
@@ -2334,29 +2333,6 @@ TableView get_sorted_view(TableRef t, std::string query_string)
 {
     Query q = t->query(query_string);
     return q.find_all();
```

Comment:
I think it's still useful to run it through serialization and then parse the output of that, just to make sure that we can always parse what it serializes to. In fact, I think that would catch that descriptors applied to the query class are not serialized?

Reply:
Yes, of course.